lua/example_of_dynamic_config_request.lua

```diff
@@ -0,0 +1,15 @@
+local ffi = require("ffi")
+
+-- TODO Add support for other operations. For example:
+-- Functions to add/del/list the GK FIB entries.
+-- Functions to list the ARP table.
+-- Functions to list the ND table.
+-- Functions to process the GT policies.
+-- ......
+
+-- The client's Lua script should contain code to
+-- call the corresponding functions defined below.
```

Comment:
Just to confirm, the purpose of this file is to maintain the functions for processing clients' requests. Later we may add functions for processing GK FIB entries, etc. Don't we need a function name for processing different requests?